website/src/app/views/apply/Apply.js

```diff
@@ -1,10 +1,38 @@
 import React from 'react';
+import { animated, useSpring } from 'react-spring';
+import applyGraphic from '../../../images/apply/apply.svg';
+import animationConfig from '../animationConstants';
 import './Apply.css';
 
 function Apply() {
+  const slideUp = useSpring(animationConfig.slideUp(true));
   return (
     <main>
-      {/* Content here */}
+      <animated.div style={slideUp} className="top-panel">
+        <div className="top-panel-text">
+          <div className="inside-top-panel-text">
+            <h1 className="top-panel-title">Why join CTC?</h1>
+            <p className="top-panel-description">
+              Commit the Change is devoted to contributing to the community
+              by creating software products for non-profits. As part of this,
+              our developers and designers learn how to work both within their
+              teams and across cross-functional teams in an Agile-like environment.
+              Applying to CTC is a great opportunity to be part of an organization
+              that values innovative technology being used for social good!
+            </p>
+            <a href="#bottom" className="common-pink-button">
```

Comment:
Dummy href, will probably be switched to the "Application Process" section once completed